eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/trace/OpenTelemetryTraceFactory.java

```diff
@@ -41,15 +44,18 @@
 
     private SpanExporter spanExporter;
 
-    private SpanExporter defaultExporter = new LoggingSpanExporter();
+    private SpanExporter defaultExporter = new logExporter();
 
     private SpanProcessor spanProcessor;
 
+    // Name of the service(using the instrumentationName)
+    private String SERVICE_NAME;
+
     public OpenTelemetryTraceFactory(CommonConfiguration configuration){
-        //different spanExporter
         try {
             String exporterName = configuration.eventMeshTraceExporterType;
-            spanExporter = (SpanExporter) Class.forName("io.opentelemetry.exporter."+exporterName+"."+exporterName+"SpanExporter").newInstance();
+            //todo different spanExporter
```

Comment:
I have no idea how to set different exporter.What should I do?

Reply:
Different exporter may implement the `SpanExporter` and you can use the reflection to get different exporter instance